core/events/bus.go

```diff
@@ -202,6 +203,7 @@ var (
 		eventspb.BusEventType_BUS_EVENT_TYPE_TRANSACTION_RESULT:               TransactionResultEvent,
 		eventspb.BusEventType_BUS_EVENT_TYPE_SNAPSHOT_TAKEN:                   CoreSnapshotEvent,
 		eventspb.BusEventType_BUS_EVENT_TYPE_PROTOCOL_UPGRADE_DATA_NODE_READY: ProtocolUpgradeDataNodeReadyEvent,
+		eventspb.BusEventType_BUS_EVENT_TYPE_DISTRESSED_ORDERS_CLOSED:         DistressedOrdersClosedEvent,
```

Comment:
see line 208 :)

Reply:
Damn... Can't believe I missed that, will add now

Reply:
done